modules/react/src/components/export-video/export-video-panel-container.js

```diff
@@ -148,15 +152,24 @@ export class ExportVideoPanelContainer extends Component {
     let newWidth = encoderSettings.gif.width;
     let newHeight = encoderSettings.gif.height;
 
-    if (resolution === 'Good (540p)') {
+    if (resolution === 'Good 16:9 (540p)') {
```

Comment:
To avoid small spelling mistakes in the future, I think it'd be great to add all of these constants into a file.

`export const GOOD_16_9 = 'Good 16:9 (540p)';` kind of thing.

Reply:
Good idea! It would make it a lot easier to compare. Will add it as a task. To be honest though I'm not a fan of 6 drop downs. Would it be better to do something like what Kepler does and obscure options based on whether 4:3 or 16:9 is picked?

![image](https://user-images.githubusercontent.com/33266041/101111757-b9945180-3590-11eb-81f3-d8c5135fb0f6.png)

Reply:
Yes I think we should mimic that UI, which we can call a task of its own.

I consider string literals like this an anti-pattern called "magic strings", so going forward we should always define constants for random strings used in control flows.